src/store/user.js

```diff
@@ -29,6 +29,7 @@ const actions = {
   logout(context) {
     const logoutFinalizer = () => {
       context.dispatch('cleanUpLoginState')
+      context.dispatch('loadSettingsValues')
```

Comment:
Why load the settings during logout?

Reply:
Because there will be settings that exist without user context. For example, when we provide multiple themes in the future, an admin could set a default theme in settings. That has to be available even if you don't have a user context. So as a result, all changes to the authenticated user (changing to another one or signing out) need to trigger a settings reload.

Reply:
I see. I am just a bit afraid of relying on authentication in this case because of the public link's context.